mantle/cmd/kolet/kolet.go

```diff
@@ -271,10 +271,16 @@ func runExtUnit(cmd *cobra.Command, args []string) error {
 	// proxy it into a channel
 	rebootChan := make(chan string)
 	errChan := make(chan error)
-	err := exec.Command("mkfifo", rebootRequestFifo).Run()
-	if err != nil {
-		return err
+	// Check if FIFO file already exists to prevent mkfifo from failing for NonExclusive tests
+	// since each test will try to create a new FIFO on the same machine
+	// NOTE: we don't expect that NonExclusive tests will need to use "rebootRequestFifo"
```

Comment:
```suggestion
	// NOTE: we don't expect that NonExclusive tests will need to use "rebootRequestFifo" 
```

Is there anything that enforces that a test that tries to reboot won't be allowed to be nonexclusive?

Reply:
There is nothing that prevents a nonexclusive test from trying to reboot currently. Not sure what a good way to enforce that restriction would be.

Reply:
We should be able to leave this code alone and instead for non-exclusive machines not even try to create it since reboots shouldn't be supported.

One way to do this is to have `runExternalTest` pass a flag to `kola run-test-unit` when it calls it and then in `runExtUnit` we'd just skip over the `mkfifo`. `runExternalTest` could also touch a file, like `/run/coreos/non-exclusive` which the `autopkgtestRebootScript` and `autopkgtestRebootPrepareScript` scripts check and error out on if it exists. That way, the test will clearly error out if it tries to reboot.